.circleci/config.yml

```diff
@@ -592,8 +592,9 @@ jobs:
       - run: node ./scripts/circleci/setup-mdc-canary.js
 
       # Setup the components repository to use the MDC snapshot builds.
-      # Run project tests with the MDC canary builds.
+      # Run project tests, and build the dev-app theme with the MDC canary builds.
       - run: bazel test --build_tag_filters=-docs-package,-e2e --test_tag_filters=-e2e --build_tests_only -- src/...
+      - run: bazel build -- src/dev-app:theme
```

Comment:
Maybe instead of this we should add a test for the density APIs in the current theming tests?

Reply:
yeah, I like that better. I added one for the experimental package in the meanwhile, and fixed/prepared the stable one (to also catch issues with density)